### PR TITLE
add pending column to CSV export on wallet page, issue #887

### LINKED ISSUE #887: Wallet CSV export: add a column / field to report on "payment status"
### Situation
The wallet UI shows `payment status, memo, date, amount (sats), fee (sats)`.
The CSV export does not have a `payment status`, hence in accounting, a critical factor (paid or pending) is missing

### Proposal
Add `payment status` to the CSV export

### PR DESCRIPTION
**Pull request by dni**
Closes #887

### CODE REVIEW

**Comment by callebtc** [comment]
Please `make format` before we can merge!

**Comment by callebtc** [comment]
Two questions: What's the difference between pending and isPaid? Why isn't it part of `this.paymentsTable.columns` already?

**Comment by dni** [comment]
_What's the difference between pending and isPaid?_
they are equivalent, i remove isPaid.

_Why isn't it part of this.paymentsTable.columns already?_
 my guess is, they did not put it into paymentsTable, because they needed more control of the rendering of the column / payment details link / icon.

File: lnbits/core/static/js/wallet.js [hunk 1]
```diff
@@ -668,7 +668,14 @@ new Vue({
       })
     },
     exportCSV: function () {
-      LNbits.utils.exportCSV(this.paymentsTable.columns, this.payments)
+      let columns = this.paymentsTable.columns
```
Comment (callebtc) [inline lnbits/core/static/js/wallet.js:671]: Please add a short comment here why you're doing this, nobody will understand when looking at it in two years.

**Pull request merged**